File: c/DoWhile.c

// C program to illustrate do-while loop
#include <stdio.h>
  
int main()
{
    int i = 1; // Initialization expression
  
    do
    {
        // loop body
        printf( "Welcome to Do While loop in C %d\n", i);    
  
        // update expression
        i++;
  
    }  while (i < 5);   // test expression
  
    return 0;
}

File: programmingloops/DoWhile.java
// Java program to illustrate do-while loop
class dowhileloopDemo
{
	public static void main(String args[])
	{
		int x = 1;
		do
		{
			// The line will be printed even
			// if the condition is false
			System.out.println("Welcome to Do While Loop in Java " + x);
			x++;
		}
		while (x < 5);
	}
}
